From: Hannes Reinecke <hare@suse.de>
Subject: Enable ATAPI support for libata
Patch-Mainline: obsolete

As SATA CD-ROMs getting more and more common we really want to use them.

--- linux-2.6.14/drivers/scsi/libata-core.c.orig	2005-10-28 02:02:08.000000000 +0200
+++ linux-2.6.14/drivers/scsi/libata-core.c	2005-11-15 08:44:28.000000000 +0100
@@ -75,7 +75,7 @@ static void __ata_qc_complete(struct ata
 static unsigned int ata_unique_id = 1;
 static struct workqueue_struct *ata_wq;
 
-int atapi_enabled = 0;
+int atapi_enabled = 1;
 module_param(atapi_enabled, int, 0444);
 MODULE_PARM_DESC(atapi_enabled, "Enable discovery of ATAPI devices (0=off, 1=on)");
 
